components/flow/FlowSpPast.vue

```diff
@@ -1,61 +1,78 @@
+          <em :class="$style.underline" place="closeContact">{{ $t('濃厚接触') }}</em>
+        </i18n>
+      </template>
+      <template v-else>
+        <i18n tag="span" path="{closeContact}をした方" :class="$style.behavior">
+          <em :class="$style.underline" place="closeContact">{{ $t('濃厚接触') }}</em>
+        </i18n>
+        <span :class="$style.source">{{ $t('「新型コロナウイルス感染者」と') }}</span>
+      </template>
+    </p>
+    <div :class="$style.condition">
       <div :class="$style.item">
-        {{ $t('または') }}
+        <div class="py-4">{{ $t('発熱') }}</div>
       </div>
+      <div :class="$style.item">{{ $t('または') }}</div>
       <div :class="$style.item">
-        <div class="py-4">
-          {{ $t('呼吸器症状') }}
-        </div>
+        <div class="py-4">{{ $t('呼吸器症状') }}</div>
       </div>
     </div>
-    <div :class="['mb-5', $style.hr]" />
-    <p class="mb-5">
-      <span :class="$style.small">流行地域への渡航・居住歴がある方</span><br />
-      <span :class="$style.underline">ご本人</span>か<!--
-   --><span :class="$style.underline">濃厚接触</span>をした方
+    <div :class="$style.hr" />
+    <p :class="$style.type">
+      <template v-if="!langsWithoutFlowTitle.includes($i18n.locale)">
+        <strong :class="$style.source">{{ $t('流行地域への渡航・居住歴がある方') }}</strong>
+        <i18n tag="span" :class="$style.behavior" path="{you} か {closeContact}をした方">
+          <em :class="$style.underline" place="you">{{ $t('ご本人') }}</em>
+          <em :class="$style.underline" place="closeContact">{{ $t('濃厚接触') }}</em>
+        </i18n>
+      </template>
+      <template v-else>
+        <i18n tag="span" :class="[$style.behavior, $style.small]" path="travel history from {area}">
+          <em :class="$style.underline" place="area">{{ $t('COVID-19 prevalent area') }}</em>
+        </i18n>
+        <i18n
+          tag="span"
+          :class="[$style.source, $style.small]"
+          path="been {inCloseContact} with returnees"
+        >
+          <em :class="$style.underline" place="inCloseContact">{{ $t('in close contact') }}</em>
+        </i18n>
+      </template>
     </p>
-    <div :class="['mb-4', $style.flexContainer]">
+    <div :class="$style.condition">
       <div :class="$style.item">
-        <div class="py-4">
-          {{ $t('呼吸器症状') }}
-        </div>
-      </div>
-      <div :class="$style.item">
-        {{ $t('かつ') }}
+        <div class="py-4">{{ $t('呼吸器症状') }}</div>
       </div>
+      <div :class="$style.item">{{ $t('かつ') }}</div>
       <div :class="$style.item">
-        <div :class="['py-4', $style.block]">
-          <span :class="$style.small">発熱</span>
-          <div>
-            37.5℃以上
+        <div>
+          <div :class="$style.temp">
+            <span :class="$style.small">{{ $t('発熱') }}</span>
+            <i18n tag="span" path="{tempNum}以上">
+              <span :class="$style.temp" place="tempNum">
+                {{ $t('37.5℃') }}
+              </span>
+            </i18n>
```

Comment:
```suggestion
            <i18n tag="span" path="発熱{temperature}" :class="新規スタイル">
              <i18n tag="span" path="{tempNum}以上" place="temperature">
                <span :class="$style.temp" place="tempNum">
                  {{ $t('37.5℃') }}
                </span>
              </i18n>
            <i18n>
```
このような入れ子構造になるようにスタイル調整していただけますか？
韓国語などでは「発熱」が「37.5℃以上」の下に来ます（「37.5℃以上の発熱」）